sudoku/easy/script.js

```diff
@@ -0,0 +1,53 @@
+import { addRandom } from './addRandom.js';
+import { checkGrid, checkRowsArr, checkGridArr, checkColumnsArr } from './checkFunctionsRandom.js';
+import { start, pause, reset } from '../general/stopwatch.js';
+export let answers = [];
+export let check = document.querySelector('.check');
+let div = document.querySelectorAll('.board>div');
+export let startGamebtn = document.querySelector('.change');
+let matrix = [];
+let arr = [];
+
+for (let i = 0; i < div.length; i++) {
+  div[i].classList.add(i + 1);
+}
+////emptying
+startGamebtn.addEventListener('click', () => {
+  for (let i = 0; i < div.length; i++) {
+    div[i].innerHTML = '';
+  }
+});
+
+startGamebtn.addEventListener('click', addRandom);
+
+function split(arr, len) {
+  let i, n;
+  (i = 0), (n = arr.length);
+  while (i < n) {
+    matrix.push(arr.slice(i, (i += len)));
+  }
+  return matrix;
+}
+
+check.addEventListener('click', () => {
+  for (let i = 0; i <= 8; i++) {
```

Comment:
avoid hardcoded numbers. Put these numbers in a separate constants. In that case the code would be more readable and easier to change